cypress.config.js

```diff
@@ -16,6 +16,19 @@ module.exports = defineConfig({
             password: '12345Qwert!',
           };
         },
+        generateUserDataSettings() {
+          let randomNumber = Math.ceil(Math.random(1000) * 1000);
+          let userName = faker.name.firstName() + `${randomNumber}`;
+          return {
+            username: 'riot',
+            newUsername: userName.toLowerCase(),
+            bio: faker.lorem.words(),
+            email: 'riot@qa.team',
```

Comment:
The initial email should be generated using faker instead of hardcoding a value.